<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>573</width>
    <height>803</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="topMargin">
      <number>10</number>
     </property>
     <property name="horizontalSpacing">
      <number>7</number>
     </property>
     <property name="verticalSpacing">
      <number>30</number>
     </property>
     <item row="6" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QComboBox" name="group_cbbox"/>
       </item>
      </layout>
     </item>
     <item row="5" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QComboBox" name="project_cbbox"/>
       </item>
      </layout>
     </item>
     <item row="11" column="1">
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>15</number>
       </property>
       <item>
        <widget class="QComboBox" name="seg_cbbox"/>
       </item>
      </layout>
     </item>
     <item row="4" column="0" alignment="Qt::AlignLeft">
      <widget class="QLabel" name="owner_lbl">
       <property name="text">
        <string>Owner</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QComboBox" name="owner_cbbox"/>
       </item>
      </layout>
     </item>
     <item row="5" column="0" alignment="Qt::AlignLeft">
      <widget class="QLabel" name="project_lbl">
       <property name="text">
        <string>Project</string>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QComboBox" name="roi_cbbox"/>
       </item>
      </layout>
     </item>
     <item row="7" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QComboBox" name="ic_cbbox"/>
       </item>
      </layout>
     </item>
     <item row="7" column="0" alignment="Qt::AlignLeft">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Image Collections</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="8" column="0" alignment="Qt::AlignLeft">
      <widget class="QLabel" name="roi_lbl">
       <property name="text">
        <string>ROI</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="6" column="0" alignment="Qt::AlignLeft">
      <widget class="QLabel" name="group_lbl">
       <property name="text">
        <string>Group</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QCheckBox" name="checkBox">
       <property name="text">
        <string>Segmentations</string>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="seg_lbl">
       <property name="text">
        <string>Segmentations</string>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QCheckBox" name="checkBox_2">
       <property name="text">
        <string>Raw</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="zslices_lbl">
       <property name="text">
        <string>No. of Z slices</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QLineEdit" name="zslice_le"/>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>15</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <widget class="QPushButton" name="load_metadata_btn">
       <property name="text">
        <string>Load Metadata</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="loadn_main_btn">
       <property name="text">
        <string>Load Images/ROis</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clear_btn">
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QProgressBar" name="status_progress">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="meta_ls"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
